### DISCUSSION TITLE
Just finished the STEM Payload board v1.3.2b

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#300] [Show and tell]
I finished the first of the new beta boards for version 1.3.2 beta. All went well. I had two bad solder joints and they needed a little reheating. It passed all testing with good results. Its my first project with a Pi Pico so I found that quite interesting. Of course, the construction steps by Alan Johnston were critical to this success. I followed everything to the letter.....almost. I had some problems with the Pico software install because clicking on Assets as instructed caused the uf2 file name to disappear. Flashing the Pico with the uf2 file was surprisingly easy.  It worked well that there were some intermediate testing rather than powering up the stack for the first time. I went to Quickparts.com for the 3D printing of the frame pieces. I just received my quote today. The solar cells are back ordered to the end of the month unfortunately. I do have some left over from the first CubeSat build in 2022.  Next is the battery board. I would paste in a photo of the finished board if I knew how.

**Comment by W7DGP** [comment]
I'm commenting on my own post. Today I finished the Battery board and the v1.3.2 Solar Board. I noticed when I was soldering in the INA219 DC Current Sensor boards, the header that came with the sensor seemed to reject solder. I had in my stock some gold plated headers that were made to break off the number of pins needed. I used that to make the 6 ea 6-pin headers for the INA boards. That helped with the soldering requiring less heat and solder and a nicer looking result. Another finding was when installing the NiMH cells the cell in the single cell holder did not make contact at the negative end of the cell. I used the extra one in the 4-pack to replace the cell and all worked ok. Having the 3 boards finished and software tested is a good milestone. 
My last adventure for today was coming up short on extra-long stacking headers. My bad. I didn't order them. I used up the supply I had left over for the earlier CubeSat project. I will now await the backordered solar cells and the stacking headers. All that and the 3D printed frame pieces should arrive at the end of this month (Feb).

**Reply by alanbjohnston** [reply]
Congrats on the successful build!  We greatly appreciate the feedback and comments. 
 
Here's my replies to your comments:

1. The Assets tab for releases has been problematic.  Starting with the v1.3.2.1 release, I'm including a direct link in the instructions: https://github.com/alanbjohnston/CubeSatSim/releases/tag/v1.3.2.1 so hopefully this won't cause problems in the future
2. I've never tried QuickParts to 3D print. I'm waiting for a print from PCBWay.com who I use for my PCBs to see how it is.
3. I think you can paste in a photo just by dragging your photo into this window - give it a try - I'd love to see your boards!
4. I've never seen bad headers with the INA219 boards, although I have had issues soldering them at times.  I find a drop of liquid flux helps if solder doesn't seem to adhere.
5. Amazon doesn't show the solar panels back ordered for me.  Maybe double check the BOM (or the Amazon wishlist for this step: https://www.amazon.com/hz/wishlist/ls/YXY8BBKL69K7?ref_=wl_share)

Again, thanks for your feedback.  Let us know how the testing goes.

I'd strongly recommend updating your Pi Zero software to the latest version by logging in and typing:

`CubeSatSim/update`

Good luck!

**Comment by W7DGP** [comment]
Here are the 3 boards:
<https://1drv.ms/i/s!Aj7wTduw16hdhf4scYxYCRfXpub5uQ>
[https://1ocjwg.bl.files.1drv.com/y4mK9xfGZAIexrbgUY3yFjGSG0LH4d7i5drJovNhOmU0OrHwBYbw6DHedQRSTmDJBb69y3M6ZI_r1_WmltXeufe_pPdaVCWO80fPFFZLIdkR-uj7AiXy06p2_jDSvG490vhGMgYN6_OWydDnJ_lgGrOBq-SPnieoeq8BL9zF41qc0j1Qz3OwtBjJxQRmRS9VYfIU0ESME-aCGxUYGo3QS_DGg?width=200&height=150&cropmode=center]
[https://res.cdn.office.net/assets/mail/file-icon/png/cloud_blue_16x16.png]BatteryBoard.jpg
<https://1drv.ms/i/s!Aj7wTduw16hdhf4thKDMOf13HaoOuw>
[https://1ockwg.bl.files.1drv.com/y4mpxeRGD7wd3c4mCpho_MRWGfDrAXIhY0C6j8RKQ5f0cxE7usZG2XMc8HJejm97DCd2YVipG_U66rHWBSRptT_ndQZkDRA8XVvngYxZrgvQd8L3aZoP5gVAAJac8EE-M5CnbB6sRE7B-amv9xySmMKUiMhUrR9sQvzl-sN32Gwo0fTejGk9SlrDU2HxXf-nxzBrNRt5xgwnVlbZY1qsbF7Nw?width=200&height=150&cropmode=center]
[https://res-h3.public.cdn.office.net/assets/mail/file-icon/png/cloud_blue_16x16.png]Stempayloadboard.jpg
<https://1drv.ms/i/s!Aj7wTduw16hdhf4ubBdaauFqj6lxsQ>
[https://1ocpwg.bl.files.1drv.com/y4mtiu9AAmDwKtAJ-SHmkt3zEGeQPOsg5736HajStaq9zP8Rt4ZKLLanoVUt-i5EZuwmcZTJkltYO3dculp7xRLgXVPjEON0WRIsJBguJ_lG2FIS1oKrhDkbriExZpqTA0duyoeWkQS5MD0bxqeb44h4CXN4BX0gHc-wo4GP-KqbBTyoTUHP--8MTTHcTnL_mbGarJglTFAHwPoe1ydzbK3vQ?width=200&height=150&cropmode=center]
[https://res.cdn.office.net/assets/mail/file-icon/png/cloud_blue_16x16.png]SolarPanelBoard.jpg

I haven't added the zipties to the battery board yet. One thing I didn't try with the INA219 headers is a drop of flux. I'm sure that would work.

All the instructions for the 3 boards were very clear and assembly was easy. I like the mounting of the INA219s on the Solar board is much better than the first Cubesat Sim. I ended up using small wire to help with "blobbing" the address pads. The first one blobbed easily and the rest worked better with a small wire in between.

Remaining tasks are soldering up the solar panels, then stacking. The solar panels were ordered Feb 8 and delivery will be March 7th (I hope). Everything else came right away. I have a supply of metal m2.5 screws and nuts. Rather than the plastic screws, I'm going to use these.
Good point about software updates. I will do that on next power up.

Question: Will this version allow control by uplink? I'm looking forward to that. My understanding is that it will be able to receive a carrier on some frequency that will be received and can control the mode. I don't see that in the testing but I could be missing something.

73, Larry W7DGP
________________________________

**Comment by W7DGP** [comment]
Ok, dragging in the photos didn't work. The links show that everything in my Downloads folder is actually saved in OneDrive and only links exist in Downloads and Pictures. Windows 11. I didn't set up things this way. It makes a mess of my posting. Sorry.
-Larry

**Comment by alanbjohnston** [comment]
It worked - I can see the images if I follow the links. Thanks for sharing them!

Alan

**Comment by WD6DRI** [comment]
Larry,

Congratulations and welcome to the newest version of the CubeSatSim project.

David
